conf/messages

```diff
@@ -24,6 +24,7 @@ generic.hm.revenue.and.customs = HM Revenue & Customs
 generic.hm.revenue = HM Revenue
 generic.hm.and = &
 generic.hm.customs = Customs
+site.back=Back
```

Comment:
Does this match?
You seem to be passing "generic.back" as the message key in the back buttons